psalm.xml

```diff
@@ -1,16 +1,44 @@
 <?xml version="1.0"?>
 <psalm
-    errorLevel="3"
+    errorLevel="1"
     resolveFromConfigFile="true"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
-    errorBaseline="psalm-baseline.xml"
 >
     <projectFiles>
         <directory name="src" />
         <ignoreFiles>
             <directory name="vendor" />
         </ignoreFiles>
     </projectFiles>
+
+    <issueHandlers>
+        <PossiblyUndefinedMethod>
+            <errorLevel type="suppress">
+                <referencedMethod name="NodeDefinition::children"/>
+                <file name="src/DependencyInjection/Configuration.php"/>
+            </errorLevel>
+        </PossiblyUndefinedMethod>
+        <MissingConstructor>
+            <errorLevel type="suppress">
+                <file name="src/Security/User.php"/>
+            </errorLevel>
+        </MissingConstructor>
+        <MixedReturnStatement>
+            <errorLevel type="suppress">
+                <file name="src/DanskernesDigitaleBibliotekAgencyAuthBundle.php"/>
+            </errorLevel>
+        </MixedReturnStatement>
+        <MixedInferredReturnType>
+            <errorLevel type="suppress">
+                <file name="src/DanskernesDigitaleBibliotekAgencyAuthBundle.php"/>
+            </errorLevel>
+        </MixedInferredReturnType>
+        <MixedMethodCall>
+            <errorLevel type="suppress">
+                <file name="src/DependencyInjection/Configuration.php"/>
+            </errorLevel>
+        </MixedMethodCall>
+    </issueHandlers>
```

Comment:
Why not use a baseline file for this?

Reply:
My opinion: 
Baseline is only for suppressing issues in legacy code. For new code you should either code to the Psalm level set in `psalm.xml` or use `issueHandlers` to explicitly suppress issues if this is not possible because of framework interactions or similar.